binder.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: page_md
+title: The Binder Project
+title_image: assets/logo_binder.svg
+tagline: Reproducible, sharable, interactive computing environments
+permalink: /binder
+---
+
+## What is the Binder Project?
+
+The Binder project brings the power of customisable and easy to share computing
```

Comment:
Is this spelling correct? I've only "seen customizable", is it a British thing?

Reply:
Yes customisable is the UK spelling